addons/mrp/migrations/12.0.2.0/post-migration.py

```diff
@@ -27,13 +27,28 @@ def fill_mrp_workcenter_productivity_loss_loss_id(cr):
 
 
 def fill_stock_warehouse_picking_types(env):
+    def _get_picking_type_update_values(self):
+        return {
```

Comment:
Instead of harcoding this, just call "super" (the original method stored in the variable), and return the values for the desired keys.

Reply:
and what I put inside of super? I mean `data = super(????)._get_picking_type_update_values`. We don't have a class here.

Reply:
I put super between quotes on purpose. You have to do:
```suggestion
        res = _get_picking_type_update_values._original_method(self)
        return {"pbm_type_id": res["pbm_type_id"], "sam_type_id": res["sam_type_id"]}
```